components/sections/organizations/Organization/Organization.tsx

```diff
@@ -0,0 +1,56 @@
+// React/Relay
+import React, { FC } from "react";
+import { createFragmentContainer, graphql } from "react-relay";
+import singleFragmentComponent from "../../../../lib/singleFragmentComponent";
+
+// Component
+import { Props } from "./Organization.types";
+import styles from "./Organization.scss";
+
+// External
+import TeamMember from "../TeamMember";
+
+export const Organization: FC<Props> = (props) => (
+  <div>
+    <h2 className={styles.organizationName}>{props.organization_name}</h2>
+    <div className={styles.organization}>
+      <div className={styles.leftMembers}>
```

Comment:
i think you can do this with css grid? or even flex and setting flex-wrap

Reply:
What I tried to achieve here is to have two columns of the same size with the content of the left column as width as the column and the content of the right column, wrapped on their size and aligned (or pushed) to the right to create to set of text, each of them as close as possible to its borders.
This can be for sure done with grid, I don't know how much will it change over this, but I'll give it a try later.

Reply:
With grid you will be able to remove the wrapping divs for right and left members and you can just render the members. Grid will take care of the layout.

i think that something like this:

```
.organization {
  display: grid;
  grid-template-columns: 1fr 1fr;
}
```

will do the work.

Reply:
The problem with this solution is that I need all of the the right column items to be justified to the right. I couldn't do this without differentiating between one or the other.